README.md

```diff
@@ -110,4 +128,4 @@ Unless you explicitly state otherwise, any contribution intentionally submitted
 for inclusion in this project by you, as defined in the Apache-2.0 license,
 shall be dual licensed as above, without any additional terms or conditions.
 
-[contributing]: https://rustwasm.github.io/docs/wasm-bindgen/contributing/index.html
\ No newline at end of file
+[contributing]: https://rustwasm.github.io/docs/wasm-bindgen/contributing/index.html
```

Comment:
Can you figure out what happened here? I assume some line ending change?

Reply:
It's a newline added to the end of the file, I think it should be ok.

Reply:
:/
GitHub used to show this with an icon at the end ...